iosApp/iosApp/ComponentViews/StopListRow.swift

```diff
@@ -153,28 +125,24 @@ struct StopListRow<Descriptor: View, RightSideContent: View>: View {
                 HaloSeparator()
             }
             HStack(alignment: .center, spacing: 0) {
-                HStack(alignment: .center) {
-                    if showStationAccessibility, activeElevatorAlerts > 0 {
-                        Image(.accessibilityIconAlert).tag("elevator_alert")
-                    } else if showStationAccessibility, !stop.isWheelchairAccessible {
-                        Image(.accessibilityIconNotAccessible).tag("wheelchair_not_accessible")
-                    } else {
-                        EmptyView()
-                    }
-                }
-                .accessibilityHidden(true)
-                .frame(minWidth: 28, maxWidth: 28)
-                .padding(.leading, 6)
-                if stopPlacement.includeLineDiagram {
-                    routeLine
-                } else {
-                    standaloneStopIcon
-                }
+                routeLine
                 VStack(alignment: .leading, spacing: 8) {
                     Button(
                         action: { onClick() },
                         label: {
                             HStack(alignment: .center, spacing: 0) {
+                                if showStationAccessibility, activeElevatorAlerts > 0 || !stop.isWheelchairAccessible {
+                                    HStack(alignment: .center) {
+                                        if showStationAccessibility, activeElevatorAlerts > 0 {
+                                            Image(.accessibilityIconAlert).tag("elevator_alert")
+                                        } else {
+                                            Image(.accessibilityIconNotAccessible).tag("wheelchair_not_accessible")
+                                        }
+                                    }
```

Comment:
question: @boringcactus oops, forgot to mention this - is this the expected layout for accessibility in the trip details view?

Reply:
[Yep](https://www.figma.com/design/eFiLM8xjo9hKprg93BBCtZ/App-UI-Design?node-id=22387-955051&t=iE8V3tNNZDBdLDb7-4).